src/map/battle.cpp

```diff
@@ -11950,7 +11950,7 @@ static const struct _battle_data {
 	{ "client_limit_unit_lv",               &battle_config.client_limit_unit_lv,            0,      0,      BL_ALL,         },
 	{ "land_protector_behavior",            &battle_config.land_protector_behavior,         0,      0,      1,              },
 	{ "npc_emotion_behavior",               &battle_config.npc_emotion_behavior,            0,      0,      1,              },
-	{ "companion_show_guild_emblem",        &battle_config.companion_show_guild_emblem,     1,      0,      1,              },
+	{ "companion_show_guild_emblem",        &battle_config.companion_show_guild_emblem,     0,     0,      540,            },
```

Comment:
Please use the corresponding BL constants. I'd just put BL_ALL even though not all apply.